plugins/woocommerce/client/admin/client/order-attribution-install-banner/order-attribution-install-banner.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody } from '@wordpress/components';
+import { useEffect, useCallback } from '@wordpress/element';
+import { plugins, external } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+import { getPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { useOrderAttributionInstallBanner } from './use-order-attribution-install-banner';
+import {
+	BANNER_TYPE_BIG,
+	BANNER_TYPE_SMALL,
+	BANNER_TYPE_HEADER,
+} from './constants';
+import './style.scss';
+
+const WC_ANALYTICS_PRODUCT_URL =
+	'https://woocommerce.com/products/woocommerce-analytics';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = null,
+	bannerType = BANNER_TYPE_BIG,
+	eventContext = 'analytics-overview',
+	dismissable = false,
+	badgeText = '',
+	title = '',
+	description = '',
+	buttonText = '',
+} ) => {
+	const { isDismissed, dismiss, shouldShowBanner } =
+		useOrderAttributionInstallBanner();
+
+	const onButtonClick = () => {
+		recordEvent( 'order_attribution_install_banner_clicked', {
+			path: getPath(),
+			context: eventContext,
+		} );
+	};
+
+	const getShouldRender = useCallback( () => {
+		if ( bannerType === BANNER_TYPE_HEADER ) {
+			return shouldShowBanner && isDismissed;
+		}
+
+		if ( ! dismissable ) {
+			return shouldShowBanner;
+		}
+
+		return shouldShowBanner && ! isDismissed;
+	}, [ bannerType, shouldShowBanner, isDismissed, dismissable ] );
```

Comment:
Even tho now the code is much more compact I would suggest adding some comments here and in the rest of the file to explain a bit more how it works. Especially, when it should be rendered or not.